pkg/api/install/install.go

```diff
@@ -53,15 +52,15 @@ func init() {
 	groupVersion := registeredGroupVersions[0]
 	*groupMeta = latest.GroupMeta{
 		GroupVersion: groupVersion,
-		Group:        apiutil.GetGroup(groupVersion),
-		Version:      apiutil.GetVersion(groupVersion),
-		Codec:        runtime.CodecFor(api.Scheme, groupVersion),
+		Group:        groupVersion.Group,
+		Version:      groupVersion.Version,
+		Codec:        runtime.CodecFor(api.Scheme, groupVersion.String()),
 	}
 	var versions []string
 	var groupVersions []string
 	for i := len(registeredGroupVersions) - 1; i >= 0; i-- {
```

Comment:
Sure. I'm going to make it right for the latest.GroupMeta.GroupVersions.

We still need the reverse logic because the NewDefaultRESTMapper on line 92 expect the reversed order.